app.js

```diff
@@ -1,360 +1,105 @@
-var randomCustAlki = function() {
-  return (Math.floor(Math.random() * (alki.maxCustAlki - alki.minCustAlki + 1)) + alki.minCustAlki);
-};
+    var thEl = document.createElement('th');
+    thEl.textContent = hours[i];
+    trEl.appendChild(thEl);
+  };
 
-var custThatHourAlki = function() {
-  for(var i = 0; i < hours.length; i++) {
-    alki.hourlyCustomerAlki[i] = randomCustAlki();
-  }
-  console.log(alki.hourlyCustomerAlki);
+  cookieTable.appendChild(trEl);
 };
+makeHeaderRow();
 
-custThatHourAlki();
-
-// Another option to fill hourlyCustomerAlki is by using push
-    //alki.hourlyCustomerAlki.push(randomCustAlki());
-
-// Calculate and store number of cookies purchased for a single hour and storer number of cookies sold per hour in hourlySalesAlki
-var cookiesSoldThatHourAlki = function () {
-  for(var i = 0; i < hours.length; i++) {
-    alki.hourlySalesAlki.push(alki.avgCookiesPerCustAlki * alki.hourlyCustomerAlki[i]);
-  }
-  console.log(alki.hourlySalesAlki);
-};
+PatCookiesLocation.prototype.render = function() {
+  this.calcHourlyCustomers();
+  this.calcHourlySales();
+  var cookieTable = document.getElementById('patCookiesLocationTable');
+  var trEl = document.createElement('tr');
+  var tdEl = document.createElement('td');
+  tdEl.textContent = this.locationName;
+  trEl.appendChild(tdEl);
+  // cookieTable.appendChild(trEl);
 
-cookiesSoldThatHourAlki();
+  var tdEl = document.createElement('td');
+  tdEl.textContent = Math.round(this.totalSales);
+  trEl.appendChild(tdEl);
 
-// Display the values of each Pike array as unordered lists in the browser
-alki.listSalesAlki = function() {
-  var alkiID = document.getElementById('alkiID');
   for (var i = 0; i < hours.length; i++) {
-    console.log(alki.hourlySalesAlki[i]);
-    var listItem = document.createElement('li');
-    listItem.textContent = hours[i] + ': ' + Math.round(alki.hourlySalesAlki[i]) + ' cookies';
-    alkiID.appendChild(listItem);
-  }
-};
-
-alki.listSalesAlki();
+    var tdEl = document.createElement('td');
+    tdEl.textContent = Math.round(this.hourlySalesArray[i]);
+    trEl.appendChild(tdEl);
+  };
+
+  cookieTable.appendChild(trEl);
+};
+
+  // for(var i = 0; i < hours.length; i++) {
+  //   var listItem = document.createElement('li');
+  //   listItem.textContent = hours[i] + ': ' + Math.round(this.hourlySalesArray[i]) + ' cookies';
+  //   var patCookieLocationList = document.getElementById(this.iDPat);
+  //   document.getElementById(this.iDPat).appendChild(listItem);
+  // }
+  // var totalListItem = document.createElement('li');
+  // totalListItem.textContent = 'Total: ' + Math.round(this.totalSales) + ' cookies';
+  // patCookieLocationList.appendChild(totalListItem);
+// };
+
+// All instances and calls of prototypes to calculate hourly customers, hourly sales, and then render on the page
+var firstAndPike = new PatCookiesLocation('First and Pike', 23, 65, 6.3, 'pikeID');
+firstAndPike.render();
```

Comment:
You declare a new store and then render that store for each one separately. This looks like the perfect place for a loop.